### ISSUE TITLE
More Informative External Diff Tool Error Messages

### ISSUE THREAD

**Issue by chrstinalin** [mozilla/assay#100]
>There's a debatably uninformative error message when the command doesn't exist (image, ENOENT) or is used incorrectly (No error message at all). 
>
>![image](https://github.com/user-attachments/assets/52005a20-39e0-42d3-bd56-bae41605bc9f)

**Comment by willdurand** [comment]
I changed the label because while the error message might not be clear (arguably `ENOENT` is pretty common for developers), there is still an error message. We should make it better and more clear, though.

**Comment by willdurand** [comment]
Ha, I see. I am wondering if the silent error is because there is no validation on the setting. Maybe we could force a non-empty value for this setting?

**Comment by chrstinalin** [comment]
@willdurand In the case I tested, I used `git` as the diff tool command. Since `git` does not throw an error (responds with "is not a git command"), no error message is shown on Assay's side (`openInDiffTool() in `diffController.ts`).

There's probably an argument to be made here that it's no longer Assay's responsibility at that point, but I think an information message showing "success" would be helpful so that the user at least knows something happened.

**Comment by willdurand** [comment]
Fair enough, though it's very low priority since we added a default value that reviewers will likely not change anyway.
